### PR TITLE
Fix Engagement Rate in the SearchFunnelWidgetGA4.

### PR DESCRIPTION
**Pull request by sashadoes**
## Summary

Addresses issue:

- #6724 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

**Review by techanvil — COMMENTED** [review]
Hi @sashadoes, thanks for the PR here. It fully implements the IB, but I've noticed that something was missed from the IB, and the resulting chart has an incorrect y-scale (the values should be in the region 0-100).

image

Looking into this, the suggested fix was quite complicated to explain and I felt it more straightforward to simply apply the changes.

Please review the additional commits I've made to this PR, and if you're happy with them, we can merge and move this to QA.

File: assets/js/modules/search-console/components/dashboard/SearchFunnelWidgetGA4/index.js [hunk 1]
```diff
@@ -20,6 +20,7 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { identity } from 'lodash';
```
Comment (aaemnnosttv) [inline assets/js/modules/search-console/components/dashboard/SearchFunnelWidgetGA4/index.js:23]: See above

File: assets/js/modules/analytics-4/utils/extract-dashboard-data.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ import classnames from 'classnames';
  * WordPress dependencies
  */
 import { __, sprintf, _x } from '@wordpress/i18n';
+import { identity } from 'lodash';
```
[the remark this answers is not in the document]
Comment (techanvil) [reply assets/js/modules/analytics-4/utils/extract-dashboard-data.js]: Thanks for the spot! I've moved the import to **External dependencies**.

File: assets/js/modules/search-console/components/dashboard/SearchFunnelWidgetGA4/index.js (hunk 1 above)
Comment (techanvil) [reply assets/js/modules/search-console/components/dashboard/SearchFunnelWidgetGA4/index.js:23]: This one is in the right place :)

**Review by aaemnnosttv — APPROVED** [review]
LGTM, thanks @sashadoes & @techanvil 👍

**Pull request merged**